### PR TITLE
setup inicial

### PR DESCRIPTION
**Pull request by anamaiabap**
#### What problem is this solving?

This PR is a setup inicial to create a store component

#### How to test it?

Add this line in manifest   
`"vtex.list-wallet": "0.x"`

and add the "store-wallet-list" component in your store theme

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
-  "vendor": "CHANGE_ME",
-  "name": "CHANGE_ME",
-  "version": "0.0.0",
-  "title": "CHANGE_ME",
-  "description": "CHANGE_ME",
+  "vendor": "vtex",
```
Comment (wagnerlduarte) [inline manifest.json]: The vendor needs to be `vtexbr` now, doesn't it?

Comment (anamaiabap) [reply manifest.json]: You are 100% right

File: manifest.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-  "vendor": "CHANGE_ME",
-  "name": "CHANGE_ME",
-  "version": "0.0.0",
-  "title": "CHANGE_ME",
-  "description": "CHANGE_ME",
+  "vendor": "vtexbr",
+  "name": "list-wallet",
+  "version": "0.0.1",
+  "title": "Wallet App",
+  "description": "App to manage portfolio values from the app list",
```
Comment (ConradoClark) [inline manifest.json]:
```suggestion
  "description": "App to manage portfolio values from the list app",
```

File: manifest.json [hunk 3]
```diff
@@ -1,18 +1,18 @@
 {
-  "vendor": "CHANGE_ME",
-  "name": "CHANGE_ME",
-  "version": "0.0.0",
-  "title": "CHANGE_ME",
-  "description": "CHANGE_ME",
+  "vendor": "vtexbr",
+  "name": "list-wallet",
+  "version": "0.0.1",
+  "title": "Wallet App",
```
Comment (ConradoClark) [inline manifest.json]: Just a suggestion, but wouldn't it be better to rename it to `List Wallet`?
```suggestion
  "title": "List Wallet",
```
Just so it's clear that it's about the list app and not a generic wallet app, while also removing the redundant `app` keyword.

**Review by wagnerlduarte — APPROVED** [review]

File: manifest.json (hunk 3 above)
Comment (anamaiabap) [reply manifest.json]: Great ideia
You're 100% right!!
Thanks :)

**Review by ConradoClark — APPROVED** [review]

**Review by wagnerlduarte — APPROVED** [review]

**Pull request merged**